okhttp/src/main/java/okhttp3/Cookie.java

```diff
@@ -85,7 +107,7 @@ public boolean persistent() {
 
   /**
    * Returns the time that this cookie expires, in the same format as {@link
-   * System#currentTimeMillis()}. This is {@link Long#MAX_VALUE} if the cookie is not persistent, in
+   * System#currentTimeMillis()}. This is December 31, 9999 if the cookie is not persistent, in
```

Comment:
How is calling code supposed to check against this value? Shouldn't the constant be on this type?